### ISSUE TITLE
Recent PVs link doesn't work on the first click

### ISSUE THREAD

**Issue by riipah** [VocaDB/vocadb#970]
1. Navigate to front page
2. Click "Recent PVs" in the sidebar
3. Observe you are taken to search page, "Anything" tab (should be "Songs")
4. Click "Recent PVs" again
5. Observe you are taken to "Songs" tab as expected.

Note: the same seems to apply to other links in the side bar, for example "Free download".

**Comment by ycanardeau** [comment]
The URL of [Recent PVs](https://vocadb.net/Search?onlyWithPVs=True&searchType=Song&sort=AdditionDate) on non-React pages contains `True`, which is problematic. Boolean values must be lowercase (`true` and `false`), but the `ActionLink` method kindly converts them to PascalCase (`True` and `False`). Is there any way to force Blazor to output in lowercase, or should we write an extention method for that?

> Note: the same seems to apply to other links in the side bar, for example "Free download".

This is simply because the the `tag` query param hasn't been implemented yet. The easiest way would be replace `tag=free` with `tagId=${vdb.values.freeTagId}`. Should I still implement the `tag` query param, which would be a bit cumbersome?

**Comment by ycanardeau** [comment]
The URL of [Recent PVs](https://vocadb.net/Search?onlyWithPVs=True&searchType=Song&sort=AdditionDate) on non-React pages contains `True`, which is problematic. Boolean values must be lowercase (`true` and `false`), but the `ActionLink` method kindly converts them to PascalCase (`True` and `False`). I gave up using the `ActionLink` method, and I replaced it with the anchor tag helper.

> Note: the same seems to apply to other links in the side bar, for example "Free download".

This was caused by simply because the `tag` query param hasn't been implemented yet. I just replaced `tag=free` with `tagId=${vdb.values.freeTagId}`, but should we still implement the `tag` query param, which would be a bit cumbersome?

Try hard refresh, if the page doesn't load properly.
